src/PluginsSystem/Microsoft.Performance.Toolkit.Plugins.Runtime/Registry/FileBackedPluginRegistry.cs

```diff
@@ -235,6 +235,12 @@ public async Task<IReadOnlyCollection<InstalledPluginInfo>> GetAllAsync(Cancella
             return await this.fileDistributedLock.AcquireAsync(timeout, cancellationToken);
         }
 
+        /// <inheritdoc/>
+        public Task Reset(CancellationToken cancellationToken)
+        {
+            return Task.Run(() => File.Delete(this.registryFilePath), cancellationToken);
```

Comment:
>return Task.Run(() => File.Delete(this.registryFilePath), cancellationToken); [](http://example.com/codeflow?start=12&length=77)

do we want to catch IO exceptions and throw a something like RepositoryDataAccessException instead?